include/libtorrent/kademlia/node.hpp

```diff
@@ -341,6 +263,8 @@ typedef std::map<node_id, dht_mutable_item> dht_mutable_table_t;
 
 	udp_socket_interface* m_sock;
 	counters& m_counters;
+
+	dht_default_storage m_storage;
```

Comment:
I want to finish this refactor first and then proceed to the parametrization in the same style of the torrent storage constructor function.